ckanext/switzerland/templates/emails/showcase.txt

```diff
@@ -0,0 +1,9 @@
+Liebes Portalteam von opendata.swiss,
+
+Auf {{ site_title }} gibt es einen neuen
```

Comment:
I feel like either both of these should be `opendata.swiss`, or both of them should be `{{ showcase_title }}`. One of each is a bit odd. ;)

Reply:
This comment I do not understand, but anyway I have asked the BFS about the texts and will wait with the actual merge until I have those.

Reply:
Sorry, I see now that I made a typo. I should have written `{{ site_title }}`. :)